bindings/js/api.js

```diff
@@ -271,9 +306,10 @@ var HaclWasm = (function() {
         api_obj[key_module] = {};
       }
       api_obj[key_module][key_func] = function(...args) {
-        return checkIfInitialized().then(function() {
-          return callWithProto(api_json[key_module][key_func], args);
-        });
+        if (isInitialized === false) {
+           throw 'Nope';
```

Comment:
can we have a more helpful error message?

Reply:
Sorry, I actually didn't mean to introduce this change, it was a test from another version of the file. Thanks for spotting it, I will reverse it.